src/main/java/io/bitrise/trace/step/InjectTraceTask.java

```diff
@@ -539,6 +569,48 @@ static String removeCommentedCode(final List<String> lines) {
         return stringBuilder.toString();
     }
 
+    /**
+     * Gets the index of a given substring from a given code line. Ignores String and char contents.
+     *
+     * @param text      the String that will be checked.
+     * @param searchFor the substring to look for.
+     * @return the index of the start of the substring if there is a match, {@code -1} otherwise (similarly to
+     * {@link String#indexOf(int)}).
+     */
+    static int getIndexOfFromCode(final String text, final String searchFor) {
```

Comment:
The idea is we get the index of a given substring, but only if it is not a String literal.